src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1003.UrisMustBeValid_Invalid.sarif

```diff
@@ -1,38 +1,55 @@
 {
-  "$schema": "ht%tp://json.schemastore.org/sarif-2.0.0",
```

Comment:
>ht%tp [](start = 14, length = 5)

This seems like a good test, i see that we have over-written it. was there a reason for this?

Reply:
it does appear from the expected output file that we never cuaght it to begin with. maybe thats why u removed the invalidity here.

---
In reply to: [434245686](https://github.com/microsoft/sarif-sdk/pull/1898#discussion_r434245686) [](ancestors = 434245686)

Reply:
An unintended consequence of `sarif transform`. Will revert.

---
In reply to: [434250204](https://github.com/microsoft/sarif-sdk/pull/1898#discussion_r434250204) [](ancestors = 434250204,434245686)

Reply:
Ok, I now remember the reason for this one. We want each test to exhibit exactly one class of error. In this test, we want the output to include _only_ `SARIF1003.UrisMustBeValid`. With the introduction of `SARIF1020.ReferToFinalSchema`, this line also caused a `SARIF1020` error.

HOWEVER: On further reflection, the value of keeping this line as a test case outweighs the value of uniform test output -- in this one case. So I am reverting this as you asked.

---
In reply to: [434587307](https://github.com/microsoft/sarif-sdk/pull/1898#discussion_r434587307) [](ancestors = 434587307,434250204,434245686)